checker/src/org/checkerframework/checker/initialization/InitializationAnnotatedTypeFactory.java

```diff
@@ -78,8 +78,30 @@
         Flow extends CFAbstractAnalysis<Value, Store, Transfer>>
     extends GenericAnnotatedTypeFactory<Value, Store, Transfer, Flow> {
 
-    /** Annotation constants */
-    protected final AnnotationMirror COMMITTED, FREE, FBCBOTTOM, NOT_ONLY_COMMITTED, UNCLASSIFIED;
+    /**
+     * UnknownInitialization or Raw
```

Comment:
These comments would be more useful if they were links to the respective classes.